exercise2/Remove Dates

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# remove the working directory 
+
+rm work
```

Comment:
Cool!, note spaces in filenames are a bit annoying and need to be 'escaped', so I suggest using `_`